expo-zustand-styled-components/src/navigation/index.tsx

```diff
@@ -41,7 +41,7 @@ function RootNavigator() {
 
   return (
     <>
-      <StatusBar style="auto" />
+      <StatusBar style="light" />
```

Comment:
Any reason why this is changed?

Reply:
The header has a dark background color. this change affect the statusbar color.